core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -190,6 +190,22 @@ private TableProperties() {}
           + // v123.metadata.json
           "|.*[0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){3}-[0-9a-fA-F]{12}-m[0-9]+\\.avro"; // UUID-m0.avro
 
+  /** table tag management related properties */
+  public static final String ENABLE_AUTO_CREATE_TAG = "tag.auto-create.enabled";
+
+  public static final boolean ENABLE_AUTO_CREATE_TAG_DEFAULT = false;
+
+  public static final String AUTO_CREATE_TAG_TRIGGER_PERIOD = "tag.auto-create.trigger.period";
+  public static final String AUTO_CREATE_TAG_TRIGGER_PERIOD_DEFAULT =
+      TagTriggerPeriod.DAILY.propertyName();
+
+  public static final String AUTO_CREATE_TAG_TRIGGER_OFFSET_MINUTES =
+      "tag.auto-create.trigger.offset.minutes";
+  public static final int AUTO_CREATE_TAG_TRIGGER_OFFSET_MINUTES_DEFAULT = 0;
+
+  public static final String AUTO_CREATE_TAG_FORMAT = "tag.auto-create.daily.tag-format";
+  public static final String AUTO_CREATE_TAG_FORMAT_DEFAULT = "'tag-day-'yyyyMMdd";
+
```

Comment:
Maybe `yyyy-MM-dd` is better.

Reply:
I think `yyyyMMdd` is shorter and is clear enough, compare `tag-day-20230901` and `tag-day-2023-09-01`

Reply:
I means that tag could have no prefix.

Reply:
The purpose of a prefix is to differentiate between tags and branches (since tags and branches cannot have the same name), so there are two distinct ways to distinguish branches and tags with the same name on the same day.
| ways| tag_name| branch_name |
|------|----------|--------------|
|1|20220901|branch-20220901|
|2|tag-20220901|branch-20220901|

I prefer way 2. What do you think?

Reply:
tag-20220901 is fine.  20220901 has already mark the tag as a daily tag. so  tag-day is not necessary

Reply:
Make sense to me!👍 I modified it.